docs/hugo/content/reference/_index.md

```diff
@@ -155,17 +155,18 @@ Azure Database for MySQL - Single Server is on the retirement path and is [sched
-| [FlexibleServersFirewallRule](https://azure.github.io/azure-service-operator/reference/dbformysql/v1api20210501/#dbformysql.azure.com/v1api20210501.FlexibleServersFirewallRule)                 | 2021-05-01         | v1api20210501        | v2.0.0         | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/samples/dbformysql/v1api/v1api20210501_flexibleserversfirewallrule.yaml)         |
-| [FlexibleServersFirewallRule](https://azure.github.io/azure-service-operator/reference/dbformysql/v1beta20210501/#dbformysql.azure.com/v1beta20210501.FlexibleServersFirewallRule)               | 2021-05-01         | v1beta20210501       | v2.0.0-beta.0  | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/samples/dbformysql/v1beta/v1beta20210501_flexibleserversfirewallrule.yaml)       |
+Supporting 5 resources: FlexibleServer, FlexibleServersAdministrator, FlexibleServersConfiguration, FlexibleServersDatabase, FlexibleServersFirewallRule
+
+| Resource                                                                                                                                                                           | ARM Version | CRD Version    | Supported From | Sample                                                                                                                                         |
+|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------|----------------|----------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| [FlexibleServer](https://azure.github.io/azure-service-operator/reference/dbformysql/v1api20210501/#dbformysql.azure.com/v1api20210501.FlexibleServer)                             | 2021-05-01  | v1api20210501  | v2.0.0         | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/samples/dbformysql/v1api/v1api20210501_flexibleserver.yaml)                |
+| [FlexibleServer](https://azure.github.io/azure-service-operator/reference/dbformysql/v1beta20210501/#dbformysql.azure.com/v1beta20210501.FlexibleServer)                           | 2021-05-01  | v1beta20210501 | v2.0.0-beta.0  | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/samples/dbformysql/v1beta/v1beta20210501_flexibleserver.yaml)              |
+| FlexibleServersAdministrator                                                                                                                                                       | 2022-01-01  | v1api20220101  | v2.1.0         | -                                                                                                                                              |
```

Comment:
Both the link to the docs and the link to the sample have gone

Reply:
Oh, you've dropped the preview version. Breaking change?

Reply:
No because it hasn't been released yet.